src/Liquid/StandardFilters.php

```diff
@@ -506,9 +506,7 @@ public static function slice($input, $offset, $length = null)
 		if (is_array($input)) {
 			$input = array_slice($input, $offset, $length);
 		} elseif (is_string($input)) {
-			$input = $length === null
-				? substr($input, $offset)
-				: substr($input, $offset, $length);
+			$input = mb_substr($input, $offset, $length);
```

Comment:
This is save because `$length` is nullable! 👍 
```php
function mb_substr(string $string, int $start, ?int $length, ?string $encoding): string {}
```